avatar.css

```diff
@@ -0,0 +1,41 @@
+:root {
+    --white : rgb(255,255,255,1);
+    --gray : rgb(128,128,128,1);
+}
+.avatar-container {
+    display: flex;
+}
+.items {
+    margin: 2rem;
```

Comment:
Suggestion: 
**.avatar** class will override **.items**  class. So, I guess you can remove this style and if **.items**  class is not used, you can remove it from class attribute too.

Reply:
items class i used for documentation i just forgot to remove it before committing here. I had placed all items inside a container.